### DISCUSSION TITLE
Likely ‘caching mechanism’ issue when there is a change in a data file.

### DISCUSSION THREAD

**Discussion by dmresearch15** [ropensci/targets#1198] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Objective is to create a html file from a X.Rmd file.

I have stored X.Rmd in my current directory (say, drive ‘X’) and all the necessary files including data and _targets.R are also in the current directory. I have created functions.R file and stored into ‘X:\functions’ directory. Functions.R reads all the necessary R data files stored in the ‘X’ directory.

I am running the following commands at R prompt to create a html file.
setwd("X:\\")
library(targets)
tar_visnetwork()
tar_make()

It does the work. 

But whenever there is a change in a file (say, ‘Y.Rda’) and although saved in same directory ‘X’ on the same name, but after running, the current html file does not show graph for the new data but shows the old data. But I am sure the R data file has been replaced. Any idea about this problem.

If I copy all the necessary files into a new directory, say, 'D', changes the path whenever essential (say, path in functions.R) and run the commands below: 
setwd("D:\\") 
library(targets)
 tar_visnetwork()
 tar_make()

It works well.

Any idea.

**Comment by joelnitta** [comment]
It would probably help if you post a [reproducible example (reprex)](https://books.ropensci.org/targets/help.html#write-a-reprex).  Otherwise the problem is very difficult to diagnose.
